src/extras/geometries/SphereBufferGeometry.js

```diff
@@ -30,10 +30,12 @@ THREE.SphereBufferGeometry = function ( radius, widthSegments, heightSegments, p
 	thetaStart = thetaStart !== undefined ? thetaStart : 0;
 	thetaLength = thetaLength !== undefined ? thetaLength : Math.PI;
 
+	var isSector = thetaStart > 0 || thetaStart + thetaLength < Math.PI;
```

Comment:
Nice find! That's not quite right, either, though... We don't care about the length. We just want to know if (a) is the north pole included?, or (b) is the south pole included?. I'll try again...